rackcity/views/asset_views.py

```diff
@@ -321,12 +373,24 @@ def asset_modify(request):
         except MacAddressException as error:
             failure_message += "Some mac addresses couldn't be saved. " + \
                 str(error)
+        try:
+            save_power_connections(
+                asset_data=data,
+                asset_id=existing_asset.id
+            )
+        except PowerConnectionException as error:
+            failure_message += "Some power connections couldn't be saved. " + \
+                str(error)
```

Comment:
I don't think we want to throw unknown `str(error)` into the failure message because this is stuff that might look weird to the user. I'm fine with merging this as it is, but that will just mean more work for when we try to fix up all the errors. Up to you!

Reply:
Nvm I see these errors are failure messages you construct in save_power_connections. Worth usability testing later on what these will look like in action but looks good for now

Reply:
I tried all the error messages out in postman and the ones I wrote myself look pretty and informative to me.  Have been trying to keep those clean as much as possible